Backpack-SwiftUI/Badge/Classes/BPKBadgeStyle.swift

```diff
@@ -0,0 +1,42 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+extension BPKBadge {
+    public enum Style {
+        /// The default style, will change in dark mode
+        case `default`
+        
+        /// The epmahsized style, will change in dark mode
+        case emphasis
+        
+        /// The success style, using a green background
+        case success
+        
+        /// The warning style, using a yellow background
+        case warning
+        
+        /// The destructive style, using a red background
+        case destructive
+        
+        /// The inverse style, using a white background
```

Comment:
like the comment, but not too clear what is this the inverse of.
Not sure if a comment is even needed in this enum cases, as they don't drive behaviour.
I'm thinkin if the comments should say what they should be used for instead, but that may be out of scope of this pr?